packages/loot-core/migrations/1704572023730_add_account_sync_source.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE accounts ADD COLUMN account_sync_source TEXT;
```

Comment:
Can we rename this to `external_sync_source`?

Reply:
Currently on that table the `account_id` refers to the external account id. It's not a great name. But I prefixed this one the same way so that grouped together they make a bit more sense. Ideally they would be `external_account_id` and `external_sync_source`.